src/transformers/modeling_longformer.py

```diff
@@ -705,10 +705,9 @@ def forward(
         token_type_ids=None,
         position_ids=None,
         inputs_embeds=None,
-        masked_lm_labels=None,
```

Comment:
I think `masked_lm_labels` was deleted accidently here

Reply:
No it was renamed to `labels`